recipe/meta.yaml

```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [py2k]
```

Comment:
No worries. I should have caught that when you submitted the PR to staged-recipes. I would be grateful if you could pull the py2 packages. Sorry for the extra work for you